projects/hslayers/src/components/compositions/compositions-parser.service.ts

```diff
@@ -1,24 +1,28 @@
 import * as xml2Json from 'xml-js';
+import {HttpClient} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+
```

Comment:
Remove extra whitespace lines

Reply:
I personally appreciate the separating lines between different types of imports. It makes these long import lists much more synoptic.

Reply:
I like it too as long as we have only two groups - our modules and third party modules. If theres more, then it already takes extra effort to decide which group the module would belong to, how to order the groups themselves etc.

Reply:
Ok, I see you are following https://blog.bitsrc.io/an-opinionated-styleguide-for-angular-af623d54e2b8 . Looks neat. I do still think that giving special treatment to rxjs might be too much in our project.

Reply:
Ah, thanks for finding it. I remember yet another style guide where the imports were sorted in 3 groups: 3rd-party (potentially even separated to multiple groups if they are still too many), 2nd-party (that is code from different project, but same organization) and lastly this project relative-paths imports. I usually create three groups as well, where 1. Angular and others (RxJs, moment, whatever), 2. ol, 3. hslayers.
I believe there is an agreement to keep at least local imports from hslayers and other stuff separated 🙂

Reply:
Given how tightly is rxjs intertwined with Angular, I would treat it as one of Angular packages, rather than along the ol, with which it has nothing in common, what do you think?

Reply:
Yes, why not. As long as it's not a seperate group.